src/stream-ciphers.rst

```diff
@@ -375,43 +375,43 @@ The server dutifully encrypts that message using the predicted IV
        & = E(k, IV_A \xor G)
    \end{aligned}
 
-That ciphertext, C\ :sub:`M`, is exactly the ciphertext block Alice
-would have had if her plaintext block was G. So, depending on what the
-data is, Mallory has figured out if Alice has a criminal record or not,
-or perhaps some kind of embarrassing disease, or some other issue that
-Alice really expected the server to keep secret.
+The ciphertext, C\ :sub:`M`, is exactly the ciphertext block Alice
+would have if her plaintext block was G. Depending on the
+data, Mallory found out if Alice has a criminal record or not,
+or some kind of embarrassing disease, or some other issue that
+Alice really expects the server to keep secret.
 
-Lessons learned: don't let IVs be predictable. Also, don't roll your own
-cryptosystems. In a secure system, Alice and Mallory's records probably
-wouldn't be encrypted using the same key.
+Lessons learned: do not let IVs be predictable. Also, do not roll your own
+cryptosystems. In a secure system, records of Alice and Mallory probably
+are not encrypted using the same key.
```

Comment:
Hm, this is a little funky. Let's replace

> Also, do not roll your own cryptosystems. In a secure system, records of Alice and Mallory probably are not encrypted using the same key.

> As we'll see later on in the book, just having unpredictable IVs doesn't make a system safe! This is a general argument against rolling your own cryptosystems: a high-quality implementation might have encrypted those records under different keys. Or it could use other controls, like relying on the associated data of an AEAD (which we'll see later) to make sure processes wouldn't perform cryptographic operations on behalf of Mallory that would also have worked for Alice.

Reply:
Updated